nixos/modules/services/libeufin/common.nix

```diff
@@ -11,8 +11,7 @@ libeufinComponent:
     enable = lib.mkEnableOption "libeufin core banking system and web interface";
     package = lib.mkPackageOption pkgs "libeufin" { };
     debug = lib.mkEnableOption "debug logging";
-    # TODO: find a better way to
-    openFirewall = lib.mkEnableOption "Open ports in the firewall";
+    openFirewall = lib.mkEnableOption "whether to open ports in the firewall";
```

Comment:
Sorry, I don't think I was clear enough with this one. `mkEnableOption` is a bit confusing, because it adds your string in the middle of a `Whether to enable ...` block

https://github.com/NixOS/nixpkgs/blob/e9e2212a0715e5aa36fdbfb14f80b5a4508cf3c8/lib/options.nix#L91-L105

It needs to either be reworded, overwritten with `lib.mkEnableOption "" // { description = "..."; }` or just use a normal `lib.mkOption`.